src/entities/user/user.resolver.spec.ts

```diff
@@ -1,19 +1,70 @@
+import supertest from 'supertest'
 import { Test, TestingModule } from '@nestjs/testing'
+import { SendGridModule } from '@anchan828/nest-sendgrid'
+import { MongooseModule } from '@nestjs/mongoose'
+import { GraphQLModule } from '@nestjs/graphql'
+import { INestApplication } from '@nestjs/common'
+import { join } from 'path'
 
-import { UserResolver } from './entities/user.resolver'
+import { User, UserSchema } from './user.model'
+import { UserResolver } from './user.resolver'
+import { UserService } from './user.service'
 
-describe('UserResolver', () => {
-  let resolver: UserResolver
+import { Follower, FollowerSchema } from '../follower/follower.model'
+import { FollowerService } from '../follower/follower.service'
+
+import { Following, FollowingSchema } from '../following/following.model'
+import { FollowingService } from '../following/following.service'
+
+// Queries
+import { listUsersQuery } from './user.queries'
+
+describe('User', () => {
+  let app: INestApplication
+  let userService: UserService
 
   beforeEach(async () => {
-    const module: TestingModule = await Test.createTestingModule({
-      providers: [UserResolver],
+    const moduleRef = await Test.createTestingModule({
```

Comment:
This probably should be better in a common before all tests so it doesn't connect to the db on each test but only one connection per whole run.